plugins/woocommerce/tests/php/src/Admin/API/OnboardingPluginsTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Test the API controller class that handles the onboarding plugins REST endpoints.
+ *
+ * @package WooCommerce\Admin\Tests\Admin\API
+ */
+
+namespace Automattic\WooCommerce\Tests\Admin\API;
+
+use WC_REST_Unit_Test_Case;
+use WP_REST_Request;
+
+/**
+ * OnboardingPlugins API controller test.
+ *
+ * @class OnboardingPluginsTest.
+ */
+class OnboardingPluginsTest extends WC_REST_Unit_Test_Case {
+	/**
+	 * Endpoint.
+	 *
+	 * @var string
+	 */
+	const ENDPOINT = '/wc-admin/onboarding/plugins';
+
+	/**
+	 * Set up.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		// Register an administrator user and log in.
+		$this->user = $this->factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+		wp_set_current_user( $this->user );
+	}
+
```

Comment:
Since `install_and_activate` and `install_async` endpoints pose a greater risk if the permission check fails, should we add a test to make sure all these endpoints are protected by the relevant permission checks?